src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig

```diff
@@ -13,7 +13,7 @@
         {% set keepAspectRatioOnZoom = sliderConfig.keepAspectRatioOnZoom.value %}
         {% set magnifierOverGallery = sliderConfig.magnifierOverGallery.value %}
         {% set zoomModal = sliderConfig.fullScreen.value %}
-        {% set minHeight = sliderConfig.minHeight.value %}
+        {% set minHeight = sliderConfig.minHeight.value|trim|replace({' ':''}) %}
```

Comment:
IMHO it would be better to format the input before it is saved, so please to the reformat in the config elements instead of doing it here

Reply:
Basically a good idea, but difficult to implement. There are many places in the admin backend where you have CSS distance value text fields, so you would have to integrate this in a lot of places.

It would be quicker to add a corresponding function to the text field and activate it via a prop.
You would/should then also rework the config parser, so that you can activate this prop there.

Die aktuelle Lösung ist daher vielleicht nicht die Beste, aber die einfachste.

----
Me and @NinjaArmy (we currently work in the same company) are also currently working on a distance Field, consisting of number Field and unit as a dropdown, but it's not ready yet.
Primarily I had originally intended this for the config, but then we also noticed that you could use this everywhere.
For reasons mentioned above, and because many tests probably need to be rewritten, this isn't ready yet.

Reply:
I really like the idea of having a special field for css distances, then we can use this as a workaround :+1: